pkg/tecdsa/dkg/protocol_test.go

```diff
@@ -1134,7 +1161,7 @@ func TestTssFinalize_ResultTimeout(t *testing.T) {
 		cancelCtx()
 		member.tssResultChan = make(<-chan keygen.LocalPartySaveData)
 
-		err := member.tssFinalize(ctx, receivedMessages)
+		_, err := member.tssFinalize(ctx, receivedMessages)
 
 		expectedErr := fmt.Errorf(
 			"TSS result was not generated on time",
```

Comment:
Should we add a test for the confirmation state ensuring that it fails when not all finalization messages were received?

Reply:
This is a basic inactivity check that we don't usually cover with protocol unit tests.